faucet/src/bin/faucet.rs

```diff
@@ -52,6 +54,13 @@ async fn main() {
                 .takes_value(true)
                 .help("Request limit for a single request, in SOL"),
         )
+        .arg(
```

Comment:
Couldn't we just plug ip addresses into the existing  --cap/--slice parameters.  Currently these are global but in my mind, earlier in the week, these parameters were per IP

Reply:
You mean replace the global altogether?

Reply:
Yes